app/src/main/res/values/strings.xml

```diff
@@ -412,6 +412,12 @@
     <string name="please_wait">Please wait &#8230;</string>
     <string name="logging_in">Logging in…</string>
 
+    <!--Edit Check Messages-->
+    <string name="customer_edit_check_msg">Information not edited for the customer %1$s</string>
+    <string name="group_edit_check_msg">Information not edited for the group %1$s</string>
+    <string name="payroll_edit_check_msg">Information not edited for the payroll with customer identifier %1$s</string>
+    <string name="identification_edit_check_msg">Information not edited for the identification type %1$s</string>
+
```

Comment:
@EGOR-IND Its good to check if user has changed so that unnecessary operations are not performed. But it would suggest that its ok if you dont notify user that he/she has not changed info as user would know it. Instead if user changed info **save and exit** and if not **do quiet exit**. What do you say are toasts mandatory?

Reply:
See I think toasts are good, I don't see any harm in that, it's just a simple indicatiin that they haven't change anything, I can add a dialog if you find snackbar inconvenient, the dialog can have an exit and cancel button, user can exit if they don't want any changes or they can simply press cancel and get back to the edit section. What do you think?

Reply:
I dont find toast bad but as you suggested it is better to confirm and save change if user edited or else it should show something like "You didn't change any data. Are you sure you want to exit?". It be probably better that this toast.